cli/command/formatter/displayutils.go

```diff
@@ -0,0 +1,82 @@
+package formatter
+
+import (
+	"unicode/utf8"
+
+	"golang.org/x/text/width"
+)
+
+// charWidth returns the number of horizontal positions a character occupies,
+// and is used to account for wide characters when displaying strings.
+//
+// In a broad sense, wide characters include East Asian Wide, East Asian Full-width,
+// (when not in East Asian context) see http://unicode.org/reports/tr11/.
+func charWidth(r rune) int {
+	switch width.LookupRune(r).Kind() {
+	case width.EastAsianWide, width.EastAsianFullwidth:
+		return 2
+	default:
+		return 1
+	}
+}
+
+// TruncateByWidth truncates a string to fit within maxwidth.
+// For maxwidth of 1, first char of string will return even if its width > 1.
```

Comment:
Perhaps we need some more information here what "width" is, but I don't have a good suggestion at this moment.

Reply:
Modified. Sorry for the delay.